### DISCUSSION TITLE
Starting services constantly loading

### DISCUSSION THREAD

**Discussion by IronM6n** [Chia-Network/chia-blockchain#18098] [Support]
see image this what I get after I updated

![Capture](https://github.com/Chia-Network/chia-blockchain/assets/84716917/74e3b578-8dd0-410f-b8c0-5206593a099e)

**Comment by BrandtH22** [comment]
Hey @IronM6n , as a heads up we can generally provide more thorough and timely support in our discord server (https://discord.gg/chia).

if you are updating from a version prior to 2.1.x then this is likely due to having the v1 of the blockchain database which is no longer supported. there are two ways to upgrade to the v2 db but first I would verify this is the issue by going to `~/.chia/miannet/db/` and seeing if the blockchain db has a v1 or a v2 in the name. If it is v1 then proceed with the upgrade steps. If it is not v1 then please navigate to the logs and provide us with the debug.log file from `~/.chia/mainnet/log/debug.log`.

The two methods for upgrades are based on the amount of storage space left on the drive. If you have more than ~300GB available then you can use the CLI option below. If you do not then you will need to use the torrent option:

Steps for the CLI command:
1. Stop the chia client (this can be done while chia is running but stopping the client saves from any potential issues)
2. Run the command `chia db upgrade` (you do not need the input or output paths if these are still default)
- This process will take some time and requires additional space as the v1 db is compacted into the v2 db which leaves the v1 db intact
3. Verify the config file (`~\.chia\mainnet\config\config.yaml`) has the correct value under the full_node section for `database_path: db/blockchain_v2_CHALLENGE.sqlite` (should only need to change the v1 to v2)
4. After the above completes start the chia client
5. If all loads up properly (might take upwards of 30 minutes to load) then you can safely delete the v1 db found here (`~\.chia\mainnet\db\blockchain_v1_mainnet.sqlite`)


Steps for using the torrent: 
1. Download the torrent file from https://www.chia.net/downloads/#database-checkpoint
2. Use a torrent client to download the full db
3. Unpack/reassemble the torrent file that was downloaded (on windows I used 7zip, Mac and linux have built in tools that work for this)
4. Move the db to the correct folder (`~\.chia\mainnet\db\`) and update the name to "blockchain_v2_mainnet.sqlite" (removing the date information in the name)
5. Verify the config file (`~\.chia\mainnet\config\config.yaml`) has the correct value under the full_node section for `database_path: db/blockchain_v2_CHALLENGE.sqlite` (should only need to change the v1 to v2)
6. Launch chia and wait for a bit (the height to hash and peers files need to be built so this can take upwards of 30 minutes)

**Comment by BrandtH22** [comment]
No response from OP, please reopen the ticket if this issue persists
